main/RFXtrx.h

```diff
@@ -1062,6 +1062,7 @@ SDK version 4.9
 #define sTypeTEMP9 0x9  //RUBiCSON
 #define sTypeTEMP10 0xA  //TFA 30.3133
 #define sTypeTEMP11 0xB  //WT0122
+#define sTypeTEMPBleBox 0xC //BleBox tempSensor
```

Comment:
As mentioned, do not add new types, there are enough
And you can not touch this file as it is not owned by us

Reply:
- reverted in 1b3801519 (will be squashed later so that there will be no changes to this file)
- reverted new names in RFXNames.cpp in d31896716 (will be squashed later)
- added HW type checking for naming to solve naming issues in a generic way in 5a18ae465
- alias blebox enums to existing subtypes in 8e383e5e8 and c2cf6cfa6

Also, if RFXtrx.h is not to be modified, it should be in a separate repository with a git submodule here. That's the proper way of assuring it isn't modified accidentally.

Please review 5a18ae465 carefully, since there are no unit tests I could run to verify the changes are correct. (I did manually check the modified SQL queries as much as I could).
I did make sure the changes are minimal, careful and easy to review (which is a poor substitute for unit tests and good coverage). I believe it's an excellent solution to avoiding misleading names without lots of vendor-specific product names added to the database.